### PR TITLE
Add `r-pathfindr` recipe

### PR DESCRIPTION
**Pull request by egeulgen**
I'd like to add the `r-pathfindr` recipe for our R package `pathfindR`.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by egeulgen** [comment]
@BiocondaBot please add label

**Comment by hechth** [comment]
@egeulgen did you use grayskull to create this recipe or did you write it from scratch?

**Comment by egeulgen** [comment]
@hechth I used conda skeleton and modified it

File: recipes/r-pathfindr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+# Imports: DBI, AnnotationDbi, doParallel, foreach, rmarkdown, org.Hs.eg.db, ggplot2, ggraph, ggupset, fpc, grDevices, igraph, R.utils, magick, msigdbr, KEGGREST, KEGGgraph, knitr
+# Depends: R (>= 4.0), pathfindR.data (>= 2.0)
+# Suggests: testthat (>= 2.3.2), covr, mockery
+# RoxygenNote: 7.2.3
+# VignetteBuilder: knitr
+# NeedsCompilation: no
+# Packaged: 2023-05-11 21:32:43 UTC; egeulgen
+# Author: Ege Ulgen [cre, cph] (<https://orcid.org/0000-0003-2090-3621>), Ozan Ozisik [aut] (<https://orcid.org/0000-0001-5980-8002>)
+# Repository: CRAN
+# Date/Publication: 2023-05-12 07:10:03 UTC
```
Comment (hechth) [inline recipes/r-pathfindr/meta.yaml:128]: Please remove this section.

File: recipes/r-pathfindr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    - r-fpc
+    - r-ggplot2
+    - r-ggraph
+    - r-ggupset
+    - r-igraph
+    - r-knitr
+    - r-magick
+    - r-msigdbr
+    - bioconductor-org.hs.eg.db
+    - r-pathfindr.data >=2.0
```
Comment (hechth) [inline recipes/r-pathfindr/meta.yaml:51]: How come that this can't be part of the actual package?

Comment (egeulgen) [reply recipes/r-pathfindr/meta.yaml:51]: CRAN didn’t allow updates as the package size got bigger and suggested another package for only data that’d be less frequently updated. Thus, we have pathfindR.data as a dependency

**Review by hechth — APPROVED** [review]

**Pull request merged**